syntax = "proto3";

package weather;

message WeatherRequest {
  string city = 1;
  string lang = 2;
}

message WeatherResponse {
  string city = 1;
  double temperature = 2;
  int32 humidity = 3;
  double wind_speed = 4;
  string condition = 5;
  string country = 6;
  string lang = 7;
}

service WeatherService {
  rpc GetWeather (WeatherRequest) returns (WeatherResponse);
}